docs/devel/luns.md

```diff
@@ -0,0 +1,54 @@
+Direct LUNs migration
+==============
+
+Summary
+-------
+
+VM contains disks. Some of them might be connected as a direct LUN.
+When a user wish to re-use the LUN disks, he needs them attached to the VM.
+The LUN should be used in a single place, or else, it will cause a data corruption.
```

Comment:
I think it would be ok to issue a warning when there's no scsi reservation configured (on the oVirt side) - so the user would know we can't protect from this. I don't think we can do much in this case..